renovate.json5

```diff
@@ -5,33 +5,10 @@
   dependencyDashboard: true,
   automerge: false,
   packageRules: [
-    {
-      // Those cannot be upgraded to a major version until we drop support for Node 8
-      packageNames: [
```

Comment:
There's a possibility that some of these packages will also have problems in Node 10, but I think we can wait for Renovate to create the PRs and we evaluate them one-by-one. Unless anyone thinks otherwise?

Reply:
Absolutely :+1: 

Many packages from Sindre Sorhus have dropped support for Node 10 recently, but we could check one by one.